containers/Project/index.tsx

```diff
@@ -22,7 +22,7 @@ const ProjectDetails = ({
       <PrevNextProjects allProjects={allProjects} currentProject={project} />
     </div>
   ) : (
-    <div>nope</div>
+    <></>
```

Comment:
Do we want to add a spinner here instead? If not can use the `&&` syntax instead of the ternary if and we wont need an else case like this